lib/puppet/info_service/task_information_service.rb

```diff
@@ -6,7 +6,7 @@ def self.tasks_per_environment(environment_name)
     env = Puppet.lookup(:environments).get!(environment_name)
     env.modules.map do |mod|
       mod.tasks.map do |task|
-        {:module => {:name => task.module.name}, :name => task.name}
+        {:module => {:name => task.module.name}, :name => task.name, :private => task.metadata["private"], :description => task.metadata["description"]}
```

Comment:
I think it makes sense _here_ to return `:metadata => task.metadata` and then on the puppetserver end pull out the fields we care about. Since we're now loading the metadata, which is the expensive part, we may as well just return all of it and let the caller decide what to do with it. Then if we later decide we want to add additional fields to the API response etc, we only need to change one repo.

Reply:
I went back and forth considering if it would be a good idea to load all the data, and yeah it probably makes sense to return everything here.

Reply:
Returning all metadata here does mean we will need to convert the whole data structure from ruby to clojure for each task, so we might want to double check that that doesn't add significant overhead on top of the loading/parsing.